stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 4.3.3
+version: 4.3.4
```

Comment:
Please bump the minor version of the chart.

```
version: 4.4.0
```

Reply:
Sure, thank you. Done.